test/functional/search-filter.test.js

```diff
@@ -0,0 +1,84 @@
+import app from '../../src/server/server';
+import request from 'supertest-as-promised';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import * as testUtils from '../utils/test-utils';
+import { resetDatabase } from '../../scripts/seed-database';
+
+chai.use(chaiAsPromised);
+const expect = chai.expect;
+
+describe('SearchFilter', () => {
+  let accessToken;
+
+  const searchFilterFixture = [
+    {
+      id: 2,
```

Comment:
Can you leave out the id? You can also use testUtils.deleteFixturesIfExist('SearchFilter'}) to remove them in afterEach().